### DISCUSSION TITLE
Get stack in spacy project: Categorization of emotions...

### DISCUSSION THREAD

**Discussion by SergioMarreroMarrero** [explosion/spaCy#7670] [Help: Coding & Implementations] [answered]
I am trying to reproduce the spacy project: Categorization of emotions in Reddit posts. I trying to train the pipeline using the downloaded vectors but I can't. I am following step by step the readme.md:

```
spacy train \
    configs/cnn.cfg \
    --training.vectors "assets/en_fasttext_vectors" \
    --components.textcat.model.tok2vec.embed.also_use_static_vectors true
``` 

And this is the output:

`training -> vectors   not a section value that can be overwritten`

do someone know what is happening?

Thanks in advance.

**Comment by svlandeg** [comment, marked as answer]
Hi!

The config you're trying to overwrite, is defined here: https://github.com/explosion/projects/blob/v3/tutorials/textcat_goemotions/configs/cnn.cfg

You'll see that there is no parameter `vectors` within the `[training]` block - hence the error message. What should work however, is overwriting `paths.vectors` with `--paths.vectors "assets/en_fasttext_vectors"`.

It looks like the readme file you followed, has gone slightly out-of-date and we'll have to adjust it as I explained above. Thanks for the report!

[UPDATE: You can find the adjustements you'll need to here in this PR: https://github.com/explosion/projects/pull/52/files]
